runtime/gc_api/HeapIteratorAPI.cpp

```diff
@@ -407,7 +407,12 @@ j9mm_iterate_object_slots(
 	case GC_ObjectModel::SCAN_REFERENCE_MIXED_OBJECT:
 		returnCode = iterateMixedObjectSlots(javaVM, objectPtr, object, flags, func, userData);
 		break;
-
+#if JAVA_SPEC_VERSION >= 19
+	/* TODO: ??? */
+	case GC_ObjectModel::SCAN_CONTINUATION_OBJECT:
+		returnCode = iterateMixedObjectSlots(javaVM, objectPtr, object, flags, func, userData);
+		break;
+#endif /* JAVA_SPEC_VERSION >= 19 */
```

Comment:
Do we plan to have extra or different logic here? If not, you can add this case to group above:
....
        case GC_ObjectModel::SCAN_CLASSLOADER_OBJECT:
 	case GC_ObjectModel::SCAN_REFERENCE_MIXED_OBJECT:
#if JAVA_SPEC_VERSION >= 19
        case GC_ObjectModel::SCAN_CONTINUATION_OBJECT:
#endif /* JAVA_SPEC_VERSION >= 19 */
....

Reply:
I leave it separate here, because I am not very clear if we need any different logic for continuation  in this case, keep a room to update in future.